Fix scroll issue for file downloads (#1216)
RaspberryPiFoundation/editor-ui: CHANGELOG.md

### Fixed

- Styling issue on sidebar on mobile (#1194)

> - Styling issue preventing scrolling on the download section on mobile (#1216)

Reviewer: ```suggestion
- Styling issue preventing scrolling in the sidebar (#1216)
```

Fixed it everywhere and would specify in the sidebar for clarity
